cardano-sl.cabal

```diff
@@ -30,6 +30,12 @@ Flag with-explorer
 
   description: Build with explorer
 
+flag bench
```

Comment:
`criterion` is still being built. And @rockbmb reports that it's because `criterion` is a transitive dependency of something else. So maybe stack actually can infer that it doesn't need to build benchmarks dependencies and we don't need this flag (which is extra complication).